srml/staking/src/lib.rs

```diff
@@ -442,7 +445,7 @@ type ExpoMap<T> = BTreeMap<
 	Exposure<<T as system::Trait>::AccountId, BalanceOf<T>>
 >;
 
-pub trait Trait: system::Trait + session::Trait {
+pub trait Trait: system::Trait + session::Trait + timestamp::Trait {
```

Comment:
we should probably traitify timestamp and wire it in as a type rather than a module-level dependency.

Reply:
done